### PR TITLE
V2 Release cleanup

### PR DESCRIPTION
**Pull request by microstrain-sam**

### CODE REVIEW

**Review by dacuster — APPROVED** [review]
Couple of minor additional edits, but everything you did looks good

File: docs/docs.c [hunk 1]
```diff
@@ -40,7 +40,7 @@
 ///@li @ref mip_packet_c
 ///@li @ref mip_field_c
 ///@li @ref mip_parser_c
-///@li @ref mip::C::mip_cmd_result
+///@li @ref mip::C::mip_cmd_result "mip_cmd_result [C]"
```
Comment (msclissa) [inline docs/docs.c:43]: not sure what this does?

**Review by msclissa — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,7 +20,68 @@ REMOVED - A function/class has been removed.
...
+* `MIP_CMD_DESC_BASE_GPS_TIME_BROADCAST_NEW` &rarr; `MIP_CMD_DESC_BASE_GPS_TIME_UPDATE` \[C]
+* `CMD_GPS_TIME_BROADCAST_NEW` &rarr; `CMD_GPS_TIME_UPDATE` \[C++]
+* `mip::Packet` &rarr; `mip::PacketRef` \[C++]
+* The following definitions have been renamed in CMake:
+  * `WITH_SERIAL` &rarr; `MIP_USE_SERIAL`
+  * `WITH_TCP` &rarr; `MIP_USE_TCP`
+
+#### Changed
+* The following 2 extern functions have been changed to callbacks to better support shared libraries:
+  Supply your callbacks to `mip_interface_init`.
```
[the remark this answers is not in the document]
Comment (microstrain-sam) [reply CHANGELOG.md:59]: There shouldn't have been a colon here, fixed.

File: docs/docs.c (hunk 1 above)
Comment (microstrain-sam) [reply docs/docs.c:43]: This overrides the text of the link so it doesn't have the `mip::C::` prefix.

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,7 +20,68 @@ REMOVED - A function/class has been removed.
...
+### New Features
+### Interface Changes
+### Bug Fixes
+
+
+v2.0.0
+------
+
+### New features
+* CV7-INS support
```
[the remark this answers is not in the document]
Comment (microstrain-sam) [reply CHANGELOG.md:32]: Yes, CV7 was already supported.

**Pull request merged**